src/Esfa.Vacancy.Register.UnitTests/SearchApprenticeship/Application/GivenASearchApprenticeshipVacanciesRequestValidator/WhenValidatingMinimumRequiredFields.cs

```diff
@@ -53,9 +67,26 @@ public void AndCheckingErrorMessages(SearchApprenticeshipVacanciesRequest search
                     Errors =
                     {
                         new ValidationFailure("", "At least one of StandardCodes or FrameworkCodes is required.")
+                        {
+                            ErrorCode = ErrorCodes.SearchApprenticeships.StandardAndFrameworkCodeNotProvided
+                        }
+                    }
+                })
+                .SetName("No Frameworks or Standards present is not allowed"),
+            new TestCaseData(new SearchApprenticeshipVacanciesRequest
+                {
+                    FrameworkCodes = null
+                }, new ValidationResult
+                {
+                    Errors =
+                    {
+                        new ValidationFailure("", "At least one of StandardCodes or FrameworkCodes is required.")
+                        {
+                            ErrorCode = ErrorCodes.SearchApprenticeships.StandardAndFrameworkCodeNotProvided
+                        }
                     }
                 })
-                .SetName("No Frameworks or Standards present is not allowed")
+                .SetName("Frameworks is null is not allowed")
```

Comment:
Could it not say Empty Framework is not allowed

Reply:
That scenario is already covered elsewhere, this particular one is specifically for framework codes collection being null.

Reply:
its gone now.